libvast/vast/concept/convertible/data.hpp

```diff
@@ -318,7 +327,7 @@ caf::error convert(const list& src, To& dst, const list_type& t) {
   for (const auto& element : src) {
     const auto* rec = caf::get_if<record>(&element);
     if (!rec)
-      return caf::make_error(ec::convert_error, "no record in list");
+      return caf::make_error(ec::convert_error, ": no record in list");
```

Comment:
This seems inside out: adding such context is the job of the calling context, no?